spec.md

```diff
@@ -209,6 +209,23 @@ that contains both context and data).
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
+### property-bucket
```

Comment:
as mentioned on WG call, this could be simply named "properties" (more consistent with parallel "extensions")

Reply:
will do